app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>All Recipes</h1>
```

Comment:
Keep in mind this is the recipe `show` page so it's only going to be showing information about ONE recipe. US1 is the Ingedients `index` so it will show all the ingredients in the db, but `show` is just for one specific record